microsoft-office/2.0/admin/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Troubleshoot Alfresco Office Services
+---
+
+Use this information to troubleshoot common Alfresco Office Services issues.
+
+## Issue with protected view (Office 2016 for Windows)
+
+There is a known issue with Office 2016 for Windows, when switching from the protected view into editing mode. Microsoft Office falsely detects file changes on the server and attempts to merge local changes with server changes. After discarding a check out, Office does not reload the file from the server and still displays the version of the file that has just been discarded.
+
+We recommend that you turn off the protected view, by adding your server in the trusted sites or local intranet zone, or alternatively by deactivating the protected view in Office. You can deactivate the protected view by selecting Options, and Word Options, then Trust Center. Select Trust Center Settings and Protected View, and deactivate all settings. Save your changes.
+
+## Issue with immediate editing after checking in a file (Office 2016 for Mac)
+
+There is a known issue in Office 2016 for Mac, if you edit a file immediately after checking the file in. You'll see an error message saying that the file cannot be modified. The workaround is to wait a minute, and try the edit again.
+
+## Issue with Online Editing
+
+There is a known issue where Online Editing is not available using the 64-bit version of Internet Explorer. See [Plan browser support (SharePoint Server 2010)](https://docs.microsoft.com/en-us/previous-versions/office/sharepoint-server-2010/cc263526(v=office.14)?redirectedfrom=MSDN){:target="_blank"} for more information.
+
+## Error message: "The address is not valid" when connecting to `http://server:port/alfresco/aos`
+
+If you have installed Alfresco manually or upgraded from a previous version of Alfresco, you might not have installed the Alfresco Office Services AMP file. If that is the case, you will receive an error message "The address is not valid" when you try to connect and authenticate with the address: `http://server:port/alfresco/aos`. You will also see the following error message in the server log:
+
+```text
+Blocked a directory listing request from MS-Office. This indicates a broken MS-Office
+deployment. Please check that the ROOT and the _vti_bin webapps are deployed properly and
+reachable from the outside!
+```
+
+To fix this problem, ensure that you have installed the Alfresco Office Services AMP file, and deployed the `_vti_bin.war` file that is required for AOS to work correctly.
+
+See [Install Alfresco Office Services]({% link microsoft-office/latest/install/index.md %}) for more.
```

Comment:
This should link to `microsoft-office/2.0/...` - check for similar links in this **2.0** folder